src/controller/AbstractMdnsDiscoveryController.h

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2013-2017 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <mdns/Resolver.h>
+#include <platform/CHIPDeviceConfig.h>
+
+namespace chip {
+
+namespace Controller {
+
+constexpr uint16_t kMdnsPort = 5353;
+
+class DLL_EXPORT AbstractMdnsDiscoveryController : public Mdns::ResolverDelegate
```

Comment:
I'm not totally clear where this would be used - what's an abstract controller? Can you add a comment to clarify the name?

Reply:
Added a brief at the top of the class declaration in the header.

Reply:
Added a brief at the top of the class declaration in the header.